deployments/examples/ocis_oc10_backend/ocis/identifier-registration.yml

```diff
@@ -0,0 +1,125 @@
+---
+# OpenID Connect client registry.
+clients:
+  - id: phoenix
+    name: OCIS
+    application_type: web
+    insecure: yes
+    trusted: yes
+    redirect_uris:
```

Comment:
Needs two additional redirect_uris:
- http://ocis-bridge.owncloud.works/oidc-silent-redirect.html
- https://ocis-bridge.owncloud.works/oidc-silent-redirect.html

This is needed for the silent token refresh to work.